packages/app/src/cli/services/dev/app-events/app-watcher-esbuild.ts

```diff
@@ -53,7 +53,7 @@ export class ESBuildContextManager {
           APP_URL: this.url,
         },
         stdin: {
-          contents: extension.getBundleExtensionStdinContent(),
+          contents: extension.getBundleExtensionStdinContent().targets,
```

Comment:
This file is responsible for watching changes in JS/TS files during `dev`, with this it will ignore changes to condition files.

Reply:
So would we need to create a new esContext for the conditions file?